examples/jsm/renderers/common/nodes/NodeUniform.js

```diff
@@ -1,9 +1,9 @@
 import {
-	FloatUniform, Vector2Uniform, Vector3Uniform, Vector4Uniform,
+	ScalarUniform, Vector2Uniform, Vector3Uniform, Vector4Uniform,
 	ColorUniform, Matrix3Uniform, Matrix4Uniform
 } from '../Uniform.js';
 
-class FloatNodeUniform extends FloatUniform {
+class ScalarNodeUniform extends ScalarUniform {
```

Comment:
Maybe `NumberNodeUniform`?

Reply:
I renamed it to `NumberNodeUniform`, seems more familiar than `numeric scalar` or `scalar`